backend/api/Areas/Project/Mapping/Dispose/ProjectMap.cs

```diff
@@ -22,12 +22,17 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.AgencyId, src => src.AgencyId)
                 .Map(dest => dest.Agency, src => AgencyConverter.ConvertAgency(src.Agency))
                 .Map(dest => dest.SubAgency, src => AgencyConverter.ConvertSubAgency(src.Agency))
-                .Map(dest => dest.ProjectProperties, src => src.Properties)
+                .Map(dest => dest.Properties, src => src.Properties)
                 .Map(dest => dest.PublicNote, src => src.PublicNote)
                 .Map(dest => dest.PrivateNote, src => src.PrivateNote)
                 .Map(dest => dest.SubmittedOn, src => src.SubmittedOn)
                 .Map(dest => dest.ApprovedOn, src => src.ApprovedOn)
                 .Map(dest => dest.DeniedOn, src => src.DeniedOn)
+                .Map(dest => dest.ExemptionRequested, src => src.ExemptionRequested)
+                .Map(dest => dest.ExemptionRational, src => src.ExemptionRational)
+                .Map(dest => dest.NetBook, src => src.NetBook)
```

Comment:
The project now maintains these values instead of having to sum the properties.  This is important as the project will now have values that are correct for the moment in time they were saved.  This does introduce challenges in keeping them up-to-date.

There is still an issue/business decision that needs to be made related to historical projects.  As we probably don't want to update their financial values after the fact if a property is updated.